jobs/calculate-embeddings/TimerFunction/index.py

```diff
@@ -46,7 +46,9 @@ def get_universe_docs(universe: str, destdir: str) -> None:
         )
         logging.info("R script ran successfully\n")
     except subprocess.CalledProcessError as e:
-        logging.error("Error running R script:\n" + e.stdout)
+        error = RuntimeError("Error running R script:\n" + e.stderr)
+        logging.error(error)
+        raise error
```

Comment:
This is raised as an error as we cannot recover from a failure here. If we don't manage to get the data we need to embed, there is nothing we can do.